schemas/2017-10-01/Microsoft.PowerBIDedicated.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [ "A1", "A2", "A3", "A4", "A5", "A6" ]
+            },
+            { "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
+          ],
+          "description": "Name of the pricing tier. Possible values include: 'A1', 'A2', 'A3', 'A4', 'A5', 'A6'"
+        }
+      },
+      "description": "The SKU (pricing tier) of the Power BI capacity."
+    },
+    "CapacityProperties": {
+      "type": "object",
+      "properties": {
+        "administration": {
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "members": {
+                  "type": "array",
+                  "items": { "$ref": "#/definitions/administrator" }
+                }
+              }
+            },
+            { "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
+          ],
+          "required": [ "members" ],
+          "description": "Microsoft.PowerBIDedicated/capacities: Login names of administrators"
+        }
+      }
+    },
+    "administrator": {
+      "type": "string",
+      "minLength": 1,
+      "description": "The UPN of the Power BI capacity administrator."
+    }
+  },
+  "description": "Power BI capacities on Azure",
+  "id": "https://schema.management.azure.com/schemas/2017-10-01/Microsoft.PowerBIDedicated.json#",
+  "resourceDefinitions": {
+    "capacities": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [ "Microsoft.PowerBIDedicated/capacities" ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [ "2017-10-01" ]
+        },
+        "location": {
+          "type": "string",
+          "description": "Capacity location"
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": { "type": "string" }

```

Comment:
>"additionalProperties": { "type": "string" } [](start = 14, length = 44)

Also, is it possible to format the json elements to be consistent with other files in this repo?